client/api/src/main/java/org/opennms/velocloud/client/api/model/ProxyEvent.java

```diff
@@ -0,0 +1,172 @@
+package org.opennms.velocloud.client.api.model;
+
+import java.time.OffsetDateTime;
+import java.util.StringJoiner;
+
+import org.json.JSONObject;
+
+public class ProxyEvent {
+    private String detail;
+    private String category;
+    private String event;
+    private Integer id;
+    private OffsetDateTime eventTime;
+    private String proxyUsername;
+    private String networkName;
+    private String enterpriseName;
+    private String gatewayName;
+    private String message;
+    private String severity;
+
+    private ProxyEvent(final ProxyEvent.Builder builder) {
+        this.detail = builder.detail;
```

Comment:
same here, are any of this fields required to be not null?

Reply:
done